packages/composables/src/interceptors/notifications.ts

```diff
@@ -0,0 +1,37 @@
+import {
```

Comment:
Maybe this should be placed in default-theme, not sure if that's the right place.

Reply:
Yes, definitely, If you find that's the most default-theme-related interceptor you can place it right there. 
It parses a response from API without any helper, so it would be too much bound to the specific implementation, so in the first iteration, the more safe place is the default-theme.

So let's stay with the theme-specific interceptors